routes/patient/patientEntry2.js

```diff
@@ -0,0 +1,101 @@
+router.route('/patientEntries/:id').get((req, res) => {
+    PatientEntry
+        .findById(req.params.id)
+        .then(patientEntries => res.json(patientEntries))
+        .catch(err => res.status(400).json('Error: ' + err));
+});
+
+//route for getting entries by patient Id
+router.route('/patientEntry/:id').get((req, res) => {
+    PatientEntry
+        .find({"patientId": req.params.id})
+        .then(patientEntries => res.json(patientEntries))
+        .catch(err => res.status(400).json('Error: ' + err));
+});
+
+//route for getting all entries of 1st patient 
+router.route('/patientEntry/').get((req, res) => {
+    PatientEntry
+        .findOne()
+        .then(patientEntries => res.json(patientEntries))
+        .catch(err => res.status(400).json('Error: ' + err));
+});
+
+
+
+//patient entry adding route
+
+router.route('/patientEntry/add/:id').put((req, res) => { //patient Id
+
+    const formElement = {
+        symptom1 :req.body.symptom1,
+        symptom2 :req.body.symptom2,
+        symptom3 :req.body.symptom3,
+        symptom4:req.body.symptom4,
+        temp : req.body.temp,
+        comment : req.body.comment,
+        doctorNote : req.body.doctorNote,
+        immediateAttention : req.body.immediateAttention,
+        updateNote : req.body.updateNote};
+    PatientEntry.findById(req.params.id)
+    .then(patientEntries => {
+        patientEntries.patientEntry.push(formElement)
+
+        patientEntries.save()
+            .then(() => res.json('Patient Entry Added!'))
+            .catch(err => res.status(400).json('Error: ' + err));
+    })
+    .catch(err => res.status(400).json('Error: ' + err));
+
+});
+
+
+//update patient update note by index
+router.route('/patientEntry/update/:id').put((req, res) => {
+    
+    //const dialogInput = {updateNote : req.body.updateNote};
+    PatientEntry.findById(req.params.id)
+    //PatientEntry.find({"date": req.params.date})
+        .then(entry => {
+            entry.patientEntry[entry.patientEntry.length-1].updateNote = req.body.updateNote;
+                   entry.save()
+                .then(() => res.json('Patient Entry Note Updated!'))
+                .catch(err => res.status(400).json('Error: ' + err));
+        })
+        .catch(err => res.status(400).json('Error: ' + err));
+});
+
+
+
+
+
+
+
+
+////update route for the patient to add a note to their entry entered in last 24 hours
+//router.route('/patientEntry/update/:id').put((req, res) => {
+//    //PatientEntry.findById(req.params.id)
+//    PatientEntry.find({"date": req.params.date})
+//        .then(patientEntry => {
+//            patientEntry[9] = req.body.updateNote
+//            patientEntry.save()
+//                .then(() => res.json('Patient Entry Note Updated!'))
+//                .catch(err => res.status(400).json('Error: ' + err));
+//        })
+//        .catch(err => res.status(400).json('Error: ' + err));
+//});
+
+module.exports = router;
```

Comment:
Cant this be included in the other patientEntry.js

Reply:
i will delet it that is just a copy